packages/theme/components/SkeletonLoader/SkeletonLoader.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div v-if="repeat > 1">
+    <span
+      v-for="n in repeat"
+      :key="n"
+      class="skeleton"
+      :style="{ height, width }"
+    />
+  </div>
+  <span
+    v-else
+    class="skeleton"
+    :style="{ height, width }"
+  />
+</template>
+
+<script>
+import { defineComponent } from '@nuxtjs/composition-api';
+
+export default defineComponent({
+  name: 'SkeletonLoader',
+  props: {
+    width: {
+      type: String,
+      default: '100%',
+    },
+    height: {
+      type: String,
+      default: '1em',
+    },
+    repeat: {
+      type: Number,
+      default: 1,
+    },
+  },
+});
+
+</script>
+
+<style lang="scss">
+.skeleton {
+  display: inline-block;
+  vertical-align: middle;
+  background-color: #f5f5f5;
```

Comment:
Would it still look good if you used a color from StorefrontUI?
![obraz](https://user-images.githubusercontent.com/5359825/156526676-51f3b0e4-2ce0-405e-a07f-0a6825119572.png)
https://docs.storefrontui.io/?path=/docs/getting-started-development-guide-colors--page
```suggestion
  background-color: var(--some-color-var-from-storefront-ui);
```

Reply:
I used some color from StorefrontUI but not as a variable. I will explain that on slack.